packages/core/package.json

```diff
@@ -83,25 +88,30 @@
     "babel-plugin-styled-components": "2.0.2",
     "babel-plugin-transform-imports": "2.0.0",
     "cross-env": "7.0.3",
+    "cypress": "10.8.0",
+    "electron": "19.0.4",
```

Comment:
I believe this is what Snyk was complaining about. If it's necessary, we can probably use 20.1.3 like we do in `packages/gui/package.json`

Reply:
Rebased